src/components/ui/Chat/ChatBubble/ChatBubble.stories.tsx

```diff
@@ -40,146 +47,56 @@ _ChatBubble.story = {
-          <ModalButtonGroup
-              primaryButtons={[
-              <ModalButton
-                  label="Delete"
-                  variant="primary"
-                  closesModal
-                  onClick={redact}
-                  key='1'
-              />,
-              <ModalButton label="Cancel" variant="secondary" closesModal key='2'/>
-              ]}
-          />
-        </ModalBody>
-      </Modal>
-    )
-
-    const prevMessageSender = messages[i - 1]?.senderId;
-    const currMessageSender = m.senderId;
-    const nextMessageSender = messages[i + 1]?.senderId;
-    let showTail = true;
-    if (
-      currMessageSender && // it is a message
-      nextMessageSender && // the item after is a message
-      currMessageSender === nextMessageSender // the item after is from the same sender
-    ) {
-      showTail = false
-    }
-    let showName = true;
-    if (
-      currMessageSender && // it is a message
-      prevMessageSender && // the item before is a message
-      currMessageSender === prevMessageSender // the message before is from the same sender
-    ) {
-      showName = false
-    };
-
-    const variant = memberId === m.senderId ? 'outgoing' : 'incoming'
-    
-    return (
-      <ChatBubbleContainer createdTimestamp={new Date().toString()} actions={moreOptions} key={'chatBubble'+i.toString()}>
-        {showDiscardModal && discardModal}
-        {showRedactModal && redactModal}
-        {editMode && !redacted ?
-        <EditableChatBubble variant={variant} senderName={m.senderName} content={text} save={saveEdit} cancel={cancelEdit} showName={showName} showTail={showTail} /> :
-        <ChatBubble variant={variant} senderName={m.senderName} content={text} createdTimestamp={m.createdTimestamp} redacted={redacted} lastUpdatedTimestamp={lastUpdatedTimestamp} showName={showName} showTail={showTail} />}
-      </ChatBubbleContainer>
-    )
-  });
-  
   return (
-    <Flex css={`overflow-y: scroll; display: flex; justify-content: center; width: 100%; height: 40rem; background-color: ${bgd}`}>
-      <ul>{messageList}</ul>
+    <Flex layout='fill-space-centered' css={containerStyles}>
+      <ChatBubble
+        variant="outgoing"
+        senderName="Fred Miller"
+        content="This is an outgoing message with attachment"
+        showTail={showTail}
+        showName={showName}
+        css={bubbleStyles}
+      >
+        <MessageAttachment
+          name="Report.pdf"
+          size="23.3KB"
+          downloadUrl="https://test.com/download/Report.pdf"
+        />
+      </ChatBubble>
+      <ChatBubble
+        variant="incoming"
+        senderName="Sarah Anderson"
+        content="This is an incoming message with attachment. It is much longer. This amount of text reaches the max length and goes onto the next line."
+        showTail={showTail}
+        showName={showName}
+        css={bubbleStyles}
+      >
+        <MessageAttachment
+          style={{ margin: '10px' }}
```

Comment:
this can be removed since you have padding in Container

Reply:
I'm not sure what you mean. The ChatBubbleContainer isn't being used here. I think the padding is needed.

Reply:
[this](https://github.com/aws/amazon-chime-sdk-component-library-react/pull/258/files#diff-5c6f1e5b8cd3960086ff85e7266ef2ffe2aa6ee270a4cfa00d8fb767fcb3d8d1R74)

Reply:
resolved offline